src/Microsoft.ML.Core/CommandLine/CmdParser.cs

```diff
@@ -466,7 +466,7 @@ private static ArgumentInfo GetArgumentInfo(Type type, object defaults)
             var map = new Dictionary<string, Argument>();
             Argument def = null;
 
-            foreach (FieldInfo field in type.GetFields())
+            foreach (FieldInfo field in type.GetFields(BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Instance))
```

Comment:
>BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Instance [](start = 55, length = 68)

Cool. As mentioned in the associated issue, we might need to do something like this in one or two more places.